### ISSUE TITLE
CUETools: EAC log file encoding not always preserved

### ISSUE THREAD

**Issue by ffwema** [gchudov/cuetools.net#330]
A tiny bug: when encoding with CUETools, the EAC log encoding is not always preserved (the problem was partially solved in version 2.1.8). This [example log file](https://github.com/user-attachments/files/15565987/test.txt) (apologies for the .txt extension but for some reason the GitHub does not allow me to upload files with .log extension) is UTF-16 LE BOM encoded, however the CUETools converts it to UTF-8-BOM.
EAC encodes logs as:
- for versions < 1: locale code page ("ANSI", Windows-1251, GB2312, ...)
- for versions ≥ 1: UTF-16 LE BOM (logs containing checksums will fail verification with EAC's CheckLog.exe if their encoding is changed)

**Comment by c72578** [comment]
The `UTF-16LE` encoding of EAC log files is only preserved, if they contain a log checksum (#51). However, the attached example log files does not contain a log checksum.

**Comment by c72578** [comment]
Thanks for the additional info and posting the example log file, which contains localized `==== Log checksum`.
This needs to be fixed...

> Do you perhaps know the reason for changing the encoding of (checksum-less) log files?

The original request primarily concerned log files with checksum.

**Comment by ffwema** [comment]
Well, if you decide to preserve encoding only for logs with checksums, this would probably work as a quick temporary fix (until Andre Wiethoff releases EAC v2):
```
// Preserve original UTF-16LE encoding of EAC log files, which contain a log checksum
if (text.StartsWith("Exact Audio Copy V1.") && text.EndsWith(" ===="))
```

**Comment by c72578** [comment]
Yes, this will be fixed.

**Comment by c72578** [comment]
@ffwema A build with a fix has been prepared in #331, which also preserves the encoding of localized EAC logfiles, which contain a log checksum, like in [example log file with checksum](https://github.com/user-attachments/files/15569096/test.txt).
